frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -9,10 +9,9 @@ import {
   StackItem,
   Title,
 } from '@patternfly/react-core';
-import { ProjectSectionID } from './types';
 
 type DetailsSectionProps = {
-  id: ProjectSectionID;
```

Comment:
Does using an emum give us an a11y error? Do we want to keep the specificity in case later on the id is needed for something else other than just a plain id.

Reply:
Yes this was causing an a11y error due to multiple components using the same id (or in this case ProjectSectionID). Nice catch - it shouldn't need to be changed because we can still make them unique via `id={`${id}-title`}`.